pmd-core/pom.xml

```diff
@@ -74,6 +88,7 @@
                 <configuration>
                     <!-- overrides the configuration from parent pom: we have here no offline links yet -->
                     <offlineLinks combine.self="override"></offlineLinks>
+                    <excludePackageNames>net.sourceforge.pmd.cache</excludePackageNames>
```

Comment:
> Had to exclude pmd.cache from javadoc, as there are no classes there, but checked that subpackages get included in the javadoc.

That's strange, maybe a bug in javadoc? We have at least a package-info.java file there, so we should not run into such a problem, should we?